microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -83,7 +83,7 @@ def verify_ltp_lite(
         else:
             skip_test_list = []
 
-        if not block_device:
+        if not block_device and node.type_name() != "local":
```

Comment:
1. Why not support local? Add code comments.
2. Use isinstance to check by type, not by a string.